src/components/AvatarWithDisplayName.js

```diff
@@ -33,6 +33,9 @@ const propTypes = {
     /** Personal details of all the users */
     personalDetails: PropTypes.objectOf(participantPropTypes),
 
+    /** Uses the reports display name as title with relying on policies and personalDetails object */
```

Comment:
I'm a bit confused by this comment -- it's describing what the prop is being used for and not the prop itself. I think if we need a description of the usage, we should add it to where we are using the prop

Reply:
Oh that's right, updating it